examples/trials/network_morphism/cifar10/cifar10_pytorch.py

```diff
@@ -27,8 +27,7 @@
 import torch.optim as optim
 import torchvision
 
-sys.path.append("../")
-from network_morphism import utils
+import utils
```

Comment:
this is offical package or your local package? is it better to make it clear?

Reply:
[`utils`](https://github.com/Microsoft/nni/blob/master/examples/trials/network_morphism/utils.py) is the local package in this example. I find that original code `cifar10_pytorch.py` works fine in local mode, but it doesn't work in the remote mode or PAI cluster because NNI will only copy the files in the same folder as `config.yml`.

Reply:
how about `import .utils`?

Reply:
ok, no problem here